lms/views/helpers/_via.py

```diff
@@ -16,17 +16,25 @@ def via_url(request, document_url, content_type=None):
     :param request: Request object
     :param document_url: Document URL to present in Via
     :param content_type: Either "pdf" or "html" if known, None if not
+    :param options: Any extra options for the url
```

Comment:
`ViaClient.url_for` includes an `options` value already. This just exposes in on the helper.